### ISSUE TITLE
YAMLLoadWarning disrupting progress bar

### ISSUE THREAD

**Issue by raibosome** [minimaxir/automl-gs#19]
Trying out the example titanic dataset in a conda environment and encountered the following error very frequently such that it disrupts the tqdm progress bar.

```
/anaconda3/envs/automl-gs/lib/python3.6/site-packages/automl_gs/utils_automl.py:270:
YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default 
Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  metrics = yaml.load(f)
```

**Comment by tayiorbeii** [comment]
@raibosome Try reinstalling the package, or manually adding `Loader=` as seen here: https://github.com/minimaxir/automl-gs/blob/master/automl_gs/utils_automl.py#L126

**Comment by minimaxir** [comment]
This should be fixed in 0.2.1. Upgrade and let me know if it works!

**Comment by remykarem** [comment]
@minimaxir great, that solved it! Thanks!
